EMS/core-bundle/src/Form/Form/UserType.php

```diff
@@ -62,6 +63,17 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
         }
 
         $builder
+            ->add('expirationDate', DateTimeType::class, [
+                'required' => false,
+                'date_widget' => 'single_text',
+                'input' => 'datetime',
+                'attr' => [
+                    'class' => 'datetime-picker',
+                    'data-date-format' => 'D/MM/YYYY HH:mm:ss',
+                    'data-date-days-of-week-disabled' => '',
+                    'data-date-disabled-hours' => '',
+                ],
```

Comment:
Did you try this:
```suggestion
                'attr' => [
                    'class' => 'datetime-picker',
                    'data-date-format' => 'D/MM/YYYY HH:mm:ss',
                ],
```

Also we have a environment variable to set the date format

Reply:
Fixed